### DISCUSSION TITLE
Get CDN fields using Nodit Indexer

### DISCUSSION THREAD

**Discussion by teenu4** [aptos-labs/aptos-developer-discussions#453] [Questions] [answered]
### Discord user ID

425369746612027411

### Describe your question in detail.

Hello!

I attempted to replace the Official Aptos Indexer (https://api.mainnet.aptoslabs.com/v1/graphql) with Nodit’s Indexer for my Code Collision Hackathon project, aiming to apply for the additional bounty ($10k Aptos Indexer Challenge with NODIT). However, I discovered that Nodit doesn’t support CDN fields. I reached out to the Nodit team via this link: https://developer.nodit.io/discuss/66d4087eb442f00030de5045, and they responded that they have contacted the Aptos Foundation to request support for this field.

Since the hackathon deadline is approaching, I wanted to check if there has been any progress on this.

Thank you!

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

Indexer API

**Comment by 0xmaayan** [comment, marked as answer]
Hi! Nodit responded that CDN data will be supported in Nodit in the future, but not now.

If you really want to query asset URI or image URI, then it would be two options:
1. Query `token_uri` and then retrieve the asset URI and image URI from the `token_uri`
2. Use Aptos Labs' indexer API and you can query CDN data.
